src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figures {

```

Comment:
The class name `RightTriangle` is correct, but it seems like it's extending a class named `Figures`. According to the checklist, we should not use abstract classes to set behavior for classes. Instead, we should use interfaces. Consider creating separate interfaces for the `area()` and `draw()` methods, as they are not conceptually close to each other.